go.sum

```diff
@@ -199,6 +202,9 @@ github.com/duosecurity/duo_api_golang v0.0.0-20190107154727-539434bf0d45 h1:AbIf
 github.com/duosecurity/duo_api_golang v0.0.0-20190107154727-539434bf0d45/go.mod h1:UqXY1lYT/ERa4OEAywUqdok1T4RCRdArkhic1Opuavo=
 github.com/dustin/go-humanize v1.0.0 h1:VSnTsYCnlFHaM2/igO1h6X3HA71jcobQuxemgkq4zYo=
 github.com/dustin/go-humanize v1.0.0/go.mod h1:HtrtbFcZ19U5GC7JDqmcUSB87Iq5E25KnS6fMYU6eOk=
+github.com/eapache/go-resiliency v1.1.0/go.mod h1:kFI+JgMyC7bLPUVY133qvEBtVayf5mFgVsvEsIPBvNs=
```

Comment:
And these?

Can you try reverting your changes to `go.sum` entirely and then running `go test ./...` to see if they come back? `go.sum` can be somewhat hinky.

Reply:
I ran these commands on a different machine and got the same results.
```sh
> git checkout HEAD~1 go.sum
> git reset
> go clean -modcache
> GO111MODULE=on go test ./...
> GO111MODULE=on go mod tidy
```
Maybe you can reproduce the same results on your machine? The only real change I made was updating `opencensus-go` from v0.19.1 to v0.19.2 in `go.mod`.

Reply:
Yup.